test/hotspot/jtreg/compiler/c2/irTests/TestAutoVectorization2DArray.java

```diff
@@ -29,7 +29,9 @@
  * @test
  * @bug 8279258
  * @summary Auto-vectorization enhancement for two-dimensional array operations
- * @requires (os.arch != "x86" & os.arch != "i386") | vm.opt.UseSSE == "null" | vm.opt.UseSSE >= 2
+ * @requires ((os.arch == "x86" | os.arch == "i386") & (vm.opt.UseSSE == "null" | vm.opt.UseSSE >= 2))
```

Comment:
Did you check that still works on other arches? e.g.: x64/aarch64

Reply:
> Did you check that still works on other arches? e.g.: x64/aarch64

Hi, thanks for the review.
I tested it under x86_64 GNU/Linux and aarch64 GNU/Linux, and both passed the test normally.